### ISSUE TITLE
options.request_timeout is not working in KurentoClient.js

### ISSUE THREAD

**Issue by mfaisalpasha** [Kurento/bugtracker#419]
**KMS Version**: 6.12.0

Paste output of `kurento-media-server -v` in the machine where KMS is running

**Ubuntu Version**

Paste output of `lsb_release -a` in the machine where KMS is running

**Other libraries versions**:

Paste output of executing `dpkg -l | grep -Pi 'kurento|gst.*1.5|nice'` in the
machine where KMS is running

**Client libraries**

- Language: Node.js
- Version:  10.15.3

**Browsers tested**

Add OK or FAIL, along with the version, after browsers where you have tested
this issue:

- Chrome:
- Firefox:
- Safari:
- Native library:

**System description:**

Please describe your setup (where is KMS located, where are the clients,
whether you are using STUN/TURN, etc.)

**What steps will reproduce the problem?**

1. I wanted to pass request_timeout as option in the function (The function is copied from kurento docs website)
2.
3.

**What is the expected result?**

**What happens instead?**

**Does it happen with one of the tutorials?**

**Please provide any additional information below**

**Comment by j1elo** [comment]
Seems you are passing the setting correctly, so I'm not sure why it wouldn't apply it. I've followed the `options.request_timeout` variable in the [KurentoClient.js](https://github.com/Kurento/kurento-client-js/blob/master/lib/KurentoClient.js#L266) code, all the way down to [kurento-jsonrpc-js//index.js](https://github.com/Kurento/kurento-jsonrpc-js/blob/master/lib/index.js#L628)

After [this](https://github.com/Kurento/kurento-jsonrpc-js/blob/master/lib/index.js#L629) `setTimeout`, the `timeout` function should be called when the time expires, and you should see [this](https://github.com/Kurento/kurento-jsonrpc-js/blob/master/lib/index.js#L657) error message: *Request has timed out*. Is this message always happening after 20 seconds, regardless of what you configure in `options.request_timeout`?

**Comment by mfaisalpasha** [comment]
Yes. It is always happening after 20 seconds. To check, I even changed the `BASE_TIMEOUT` of KurentoClient.js to 3000. But the message is always after 20 seconds. `failAfter` is working correctly. If I pass `const options = { failAfter: 2 };` then it is happening as intended.

**Comment by 007souvikdas** [comment]
I am also facing the same issue, I changed `request_timeout` to 5 seconds,  but it's still taking 20 seconds
